### PR TITLE
Move nodes of uncomplete networks in network area diagram SVG

### PR DESCRIPTION
**Pull request by massimo-ferraro**
**Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**
no

**What kind of change does this PR introduce?**
feature

**What is the current behavior?**
Some nodes and edges cannot be correctly moved in incomplete network area diagram SVGs: the movements of half edges is not handled

**What is the new behavior (if this is a feature change)?**
Nodes and edges can be correctly moved in incomplete network area diagram SVGs: the movements of half edges is handled translating the half edges position

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [X] No

### CODE REVIEW

File: src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts [hunk 1]
```diff
@@ -1291,4 +1305,38 @@ export class NetworkAreaDiagramViewer {
             pathElement.setAttribute('d', path);
         }
     }
+
+    private translateEdge(
+        edgeNode: SVGGraphicsElement,
+        offset: Point,
```
Comment (flo-dup) [inline src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts:1311]: mousePosition

File: src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts [hunk 2]
```diff
@@ -1291,4 +1305,38 @@ export class NetworkAreaDiagramViewer {
...
+            (transform?.matrix.f ?? 0) + translation.y
+        );
+        edgeNode.setAttribute(
+            'transform',
+            'translate(' +
+                totalTranslation.x.toFixed(2) +
+                ',' +
+                totalTranslation.y.toFixed(2) +
+                ')'
+        );
```
Comment (flo-dup) [inline src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts:1330]: this could use the new `moveSvgElement` method?

**Review by flo-dup — CHANGES_REQUESTED** [review]

File: src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts (hunk 1 above)
Comment (massimo-ferraro) [reply src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts]: fixed

File: src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts (hunk 2 above)
Comment (massimo-ferraro) [reply src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts]: Code changed to using moveSvgElement method

**Review by flo-dup — APPROVED** [review]

**Pull request merged**